nixos/tests/all-tests.nix

```
@@ -72,6 +72,10 @@ in
   cockroachdb = handleTestOn ["x86_64-linux"] ./cockroachdb.nix {};
   collectd = handleTest ./collectd.nix {};
   consul = handleTest ./consul.nix {};
+  containers-next = handleTest ./containers-next.nix {};
+  containers-next-imperative = handleTest ./containers-next-imperative.nix {};
+  container-migration = handleTest ./container-migration.nix {};
```

```suggestion
  containers-migration = handleTest ./containers-migration.nix {};
```

everything else is also called containers. 🙃